lib/Config.js

```diff
@@ -1298,7 +1298,10 @@ class Config extends EventEmitter {
         }
 
         if (config.redis) {
-            this.redis = parseRedisConfig(config.redis);
+            // Fail fast to make sure we detect any bad config
+            throw new Error(
+                '[S3C-10107 unification] config.redis is not used. It should be config.utapi.redis or config.localCache'
```

Comment:
would appear in logs, should not mention ticket:
```suggestion
                'config.redis is not supported anymore: it should be config.utapi.redis or config.localCache'
```